composer.json

```diff
@@ -7,7 +7,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=5.5",
-        "google/auth": "^1.2.0",
+        "google/auth": "^1.16.0",
```

Comment:
Just a reminder that depending on the order these will be merged there could be a conflict here: https://github.com/googleapis/gax-php/pull/325/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R10

Reply:
Good idea, might as well update it to `1.17.0` then!